tests/dummy/public/test-forms/basic-fields/form.ttl

```diff
@@ -34,6 +34,28 @@ fields:8e24d707-0e29-45b5-9bbf-a39e4fdb2c11 a form:PropertyGroup;
     sh:name "Overview of all basic form fields" ;
     sh:order 1 .
 
+##########################################################
+# Heading
+##########################################################
+fields:2bb6c63d-2cb7-4e82-839f-5c1e0a4f6a5e a form:Field;
+    mu:uuid "2bb6c63d-2cb7-4e82-839f-5c1e0a4f6a5e" ;
+    sh:name "Heading";
+    form:help """ This is a 
+    <a href="https://lokaalklimaatpact.be/formulieren.html" target="_blank">link</a>. <br>
+    <ul  style="list-style-type:disc; margin-left: 30px;">
+  <li>Coffee</li>
+  <li>Tea</li>
+  <li>Milk</li>
+</ul>
+    """ ;
+    sh:order 190;
+    form:options """{ "level": 1, "skin": 6}""";
```

Comment:
oh, right, the values should be strings, or at least the level one. Appuniversum uses strict equals checks for the level so it will always default to `1` if it doesn't match  any of the strings.

https://github.com/appuniversum/ember-appuniversum/blob/development/addon/components/au-heading.hbs

It's a bit of a strange API I guess, but that's the way it is implemented at the moment 😄. `skin` is fine since it does a `==` check so js converts the number into a string first. But it might be nice to still convert them to strings anyway. I'll push the change since I have it open anyways.